api/gatewayserver.proto

```diff
@@ -104,4 +116,15 @@ service Gs {
       get: "/gs/gateways/{gateway_id}/connection/stats"
     };
   };
+
+  // Get statistics about gateway connections to the Gateway Server of a batch of gateways.
+  // This is not persisted between reconnects.
+  // Gateways that are not connected or are part of a different cluster are ignored.
+  // It's upto the client to make sure that the gateways are in the requested cluster.
+  rpc BatchGetGatewayConnectionStats(BatchGetGatewayConnectionStatsRequest) returns (BatchGetGatewayConnectionStatsResponse) {
+    option (google.api.http) = {
+      post: "/gs/gateways/connection/stats"
```

Comment:
Did we blacklist the gateway ID `connection`?

Reply:
That's a good one.